src/traces/sankey/attributes.js

```diff
@@ -127,6 +122,8 @@ module.exports = overrideAll({
             role: 'style',
             description: 'Sets the thickness (in px) of the `nodes`.'
         },
+        hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {flags: []}),
```

Comment:
Hmm, if these have no flags they should probably not be `flaglist` attributes. I realize that at some point we will likely implement flags but in the meantime they should probably be `enumerated`. Also `plotAttrs.hoverinfo` is `arrayOk`, does that work here?

Reply:
to be clear: if `arrayOk` is not supported, you don't need to add it now but we should take it out of the attribute spec.

Reply:
`arrayOk` is not supported right now. I set it to `false` for the time being.